voskcli/transcribe.py

```diff
@@ -122,7 +158,20 @@ def write_webvtt_captions(rec_results):
 
 
 def transcribe(inputFile, outputFile, model):
-
+    '''
+    Produce transcription.
+    
+    Create transcription data from inputFile,
+    process data with `write_webvtt_captions` function,
+    and save finished transcription to outputFile.
+    
+    :param inputFile: Path to input file directory
+    :type inputFile: str
+    :param outputFile: Path to output file directory
```

Comment:
Isn't this the path to the files, instead of to the directories they reside in?

```suggestion
    :param inputFile: Path to input file
    :type inputFile: str
    :param outputFile: Path to output file
```